scurvysin/__init__.py

```diff
@@ -64,6 +65,12 @@ def expfl(self) -> list:
             return ['install']
 
 
+class PiPArgParser(argparse.ArgumentParser):
+    def __init__(self):
+        super().__init__()
+        self.add_argument("-r", dest="requirement_file", help="Path to pip requirements file.")
```

Comment:
Shouldn't we support `--requirement` too?

Reply:
Definitely, I'll add it

Reply:
Added, without unit tests so far, would add after #14